pkg/reconciler/trigger/resources/secret.go

```diff
@@ -0,0 +1,35 @@
+package resources
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"knative.dev/eventing/pkg/apis/eventing/v1beta1"
+	"knative.dev/pkg/kmeta"
+)
+
+const (
+	ScaleTriggerSecretName   = "rabbitmq-scale-trigger-secret"
```

Comment:
I just realized we are going to need one of these secrets per RabbitMqBroker instance :-)